tor/src/main/scala/org/bitcoins/tor/config/TorAppConfig.scala

```diff
@@ -71,6 +72,15 @@ case class TorAppConfig(private val directory: Path, private val confs: Config*)
       None
     }
   }
+
+  lazy val enabled: Boolean = socks5ProxyParams.isDefined || torParams.isDefined
```

Comment:
This is a `||` not `&&` ? My understanding (which probably is wrong) is we need _both_ these settings.

Reply:
No this should be `||` we are using tor if either of these are enabled